frontend/template-partials/views/404.html

```diff
@@ -9,13 +9,13 @@
           <div class="govuk-grid-column-two-thirds">
             <h1 class="govuk-heading-l">{{title}}</h1>
               <p class="govuk-body">
-              {{paragraph1}}
+                If you typed the web address, check it is correct.
```

Comment:
I believe rather than adding the content directly, you want to add it into the translation for paragraph1, paragraph2, paragraph 3

Reply:
TBH i did that previously and i thought the we follow the same approach with the 500 error page

Reply:
Translations are better because then it can be in different languages.